test/distributed/_tensor/test_embedding_ops.py

```diff
@@ -184,6 +184,49 @@ def test_sharded_embedding_rowwise(self):
             self.assertEqual(comm_mode.get_total_counts(), 1)
             self.assertEqual(comm_mode.get_comm_counts()[funcol.all_reduce], 1)
 
+    @with_comms
+    def test_multiple_embeddings_rowwise(self):
+        mesh = self.build_device_mesh()
+
+        inp = torch.randint(0, 10, (4, 4), device=self.device_type)
+        replicated_inp = DTensor.from_local(inp, mesh, [Replicate()], run_check=False)
+
+        from torch.distributed._tensor.ops.embedding_ops import _MaskPartial
```

Comment:
FYI I landed https://github.com/pytorch/pytorch/pull/131211 so you probably need to rebase and adjust the import